Check and balances.md

```diff
@@ -11,11 +11,24 @@ In this case. Delijaj is the seller and Nyomi the buyer.
 While testing the trading system on the 3<sup>rd</sup> auction, a seller decided to trad some parts of the project. 
 However, she did not own any part of it but the website let her do it anyway. The parts were at 50 token. A buyer 
 decided to acquire those parts. The exchange was made and the seller finished with a -50 on the project (like a debt).
-This shouldn't happend since the parts must be only managed by the contributors and nobody else.
+This shouldn't happend since the parts must be only managed by the contributors and nobody else, and also to prevent any scam.
```

Comment:
Actually after thought it is a normal mechanism called "short selling". You can google it but in essence it is the act of selling tokens or shares in the hopes that the price will go down. If it does you have made a sale at a higher price an can now buy tokens at the lower price to refund your "debt"... I would remove this part or maybe add it to our report that we didn't understand this concept and thus learned about markets by contributing to this project